code/modules/power/supermatter/supermatter_process.dm

```diff
@@ -138,6 +139,75 @@
 	var/next_sound = round((100 - aggression) * 5)
 	last_accent_sound = world.time + max(SUPERMATTER_ACCENT_SOUND_MIN_COOLDOWN, next_sound)
 
+/obj/machinery/power/supermatter_crystal/proc/animate_the_sm() // Start the loop
+	filters = filter(type="rays", size = clamp(power/30, 0, 125), color = SUPERMATTER_COLOUR, factor = 0.6, density = 12)
+	animate(filters[1], time = 10 SECONDS, offset = 10, loop=-1)
+	animate(time = 10 SECONDS, offset = 0, loop=-1)
+	animate(filters[1], time = 2 SECONDS, size = 80, loop=-1, flags = ANIMATION_PARALLEL)
+	animate(time = 2 SECONDS, size = 10, loop=-1, flags = ANIMATION_PARALLEL)
```

Comment:
should only need one loop = -1

Reply:
The effect ends up looking wrong when I try this, either inverse or otherwise not looping properly.